ViSearchSDK/ViSearchAPISource/BaseSearchParams.h

```diff
@@ -7,6 +7,9 @@
 //
 #import <Foundation/Foundation.h>
 
+#define kVisenzeUserAgentHeader @"X-Requested-With"
+#define kVisenzeUserAgentValue @"ViSenze-OC-SDK/1.1.1"
```

Comment:
is there a better way we can get the version number automatically, in case we forget to update this value when we release new versions in the future?